closed/custom/Main.gmk

```diff
@@ -76,7 +76,7 @@ ALL_TARGETS += debug-image
 test-image-openj9 : exploded-image
 	@+$(OPENJ9_MAKE) openj9_test_image
 ifneq ($(COMPILE_TYPE), cross)
-	$(JDK_OUTPUTDIR)/bin/java -version 2>&1 | $(TEE) $(TEST_IMAGE_DIR)/openj9/java-version.txt
+	rc=0; $(JDK_OUTPUTDIR)/bin/java -version >$(TEST_IMAGE_DIR)/openj9/java-version.txt 2>&1 || rc=$$?; $(CAT) $(TEST_IMAGE_DIR)/openj9/java-version.txt; exit "$$rc"
```

Comment:
Why the difference as compared with https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/639?
It's unclear to me how this is better, but in any case, we should be consistent.

Reply:
It's because `basics.m4` also opportunistically sets `-e` in `SHELL`. If that happens to be set, then as soon as `java -version` fails, the shell exits and we don't get to run `cat`, so the output is hidden.

Reply:
That's not necessary in Java 8 because its makefile doesn't try to pass `-e` to the shell. Though if you like, I can change the Java 8 PR to be consistent with these others anyway.

Reply:
At least on Linux, `spec.gmk` for jdk11+ says:
```
BASH_ARGS:= -o pipefail -e
```
so this isn't needed; and perhaps it's reasonable to assume `bash` in jdk8 so we could use `-o pipefail` there.

Reply:
I looked at the configuration code some more. It seems to anticipate the possibility that `pipefail` is not supported by the available version of `bash`, so we _do_ want this change (and the jdk8 change should match this).